src/main/java/entities/user_matches/GroupScore.java

```diff
@@ -0,0 +1,88 @@
+package entities.user_matches;
+
+import entities.Group;
+import entities.GroupProfile;
+import entities.User;
+import entities.UserPublicProfile;
```

Comment:
Does this import statement work? To my knowledge our entities package is capitalized.